src/js/brim/program.test.ts

```diff
@@ -243,7 +243,7 @@ describe("#hasAnalytics()", () => {
       brim
         .program("* | fork ( => cut uid, _path => cut uid ) | tail 1")
         .hasAnalytics()
-    ).toBe(true)
+    ).toBe(false)
```

Comment:
why did this flip?

Reply:
Previously, we need to have the "ts" field be included in the results to perform infinite scroll. If a query happened to use "cut", it probably cut out the ts field, so we were unable to get the next page. Our solution in this PR is more robust and doesn't care if you cut fields or not. 

The hasAnalytics check only needs to check the AST for a "Summarize" proc. It used to check for "cut" and "sort". It doesn't need to anymore. Hopefully that makes sense.

If the query has analytics, then we limit the results to 2000. If not, we infinite scroll using the head / tail query trick.